src/popover/styles.scss

```diff
@@ -46,7 +54,24 @@
   background-color: transparent;
 
   cursor: pointer;
-  border-block-end: awsui.$border-divider-list-width dashed currentColor;
+
+  /*
+    This transparent border is necessary to maintain space between the trigger and the bottom-positioned popover.
+  */
+  border-block-end: awsui.$border-divider-list-width dashed transparent;
+  @include trigger-underline-styles;
+
+  &.overflow-ellipsis {
+    /*
+      This line-height needs to be overridden because the overflow: hidden would otherwise conceal the underline styles.
+    */
+    line-height: calc(1 + #{$trigger-underline-offset} + #{awsui.$border-divider-list-width});
+  }
+
+  /* stylelint-disable selector-combinator-disallowed-list, selector-max-type, selector-max-universal */
```

Comment:
I would add a code comment explaining that the combinator actually targets the custom content